.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+<!--
+    If you are changing an existing module or resource, describe why that configuration is no longer acceptable. If you are fixing a misconfiguration, make sure to describe both the intended result and the actual (faulty) result.
+-->
+
+<!--
+    If you are adding a module or resources, describe what usage will be supported when this change is applied.
```

Comment:
I think this information belong to the documentation so I don't think it is needed to be repeated in the PR description.

Reply:
The top comment is only about changes and fixes so you write for either of these, but typically not both. I'll see if I can rearrange the text to make this clearer.

Reply:
Merged the add and change section into one. See if this read better?